.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,59 @@
+name: Docker publish
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  workflow_dispatch:
+    inputs:
+      mode:
+        description: 'release/nightly, default is nightly'
+        required: true
+        default: 'nightly'
+
+jobs:
+  publish:
+    if: github.repository == 'deepjavalibrary/djl'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
should we publish to dockerhub instead? Why using a custom ECR repo?

Reply:
This is for reading the base EMR image